dwave/optimization/generators.py

```diff
@@ -145,6 +146,69 @@ def quadratic_model(qm):
 
     return model
 
+def bin_packing(weights: numpy.typing.ArrayLike,
+                bin_capacity: float
```

Comment:
We should keep the argument names consistent with https://github.com/dwavesystems/dimod/pull/1384. IMO `bin_` is implied here, so I think just `capacity` is fine.

Reply:
All suggested changes made, thanks!